src/robotDescription.h

```diff
@@ -44,6 +44,7 @@ struct simParams {
     int cmd_line_per = 1;                              // Command line sim info output period
     bool enable_2d_sim = false;                        // Lock z and theta DOFs
     bool line_search = true;                           // Enable line search method
+    int line_search_type = 0;
```

Comment:
We should use 
```
enum LineSearchType {
    Goldstein,
    Wolfe
};
```

as the identifier here.

Reply:
Sure. This enum looks more clear.